pyresample/geometry.py

```diff
@@ -1074,11 +1131,16 @@ def _compute_bound_centers_numpy(self, proj_dict, lons, lats):
         y_is_pole = (ymax >= 90 - epsilon) or (ymin <= -90 + epsilon)
         if proj4.crs.is_geographic and x_passes_antimeridian and not y_is_pole:
             # cross anti-meridian of projection
-            xmin = np.nanmin(xarr[xarr >= 0])
-            xmax = np.nanmax(xarr[xarr < 0]) + 360
-        return xmin, ymin, xmax, ymax
+            if antimeridian_mode == "global_extents":
```

Comment:
This is the only option that doesn't have an imperative mode, so should it be `use_global_extents`?